bin/citrea/tests/bitcoin_e2e/tests/mempool_accept.rs

```diff
@@ -0,0 +1,56 @@
+use async_trait::async_trait;
+use bitcoin_da::service::FINALITY_DEPTH;
+use bitcoincore_rpc::RpcApi;
+
+use crate::bitcoin_e2e::config::BitcoinConfig;
+use crate::bitcoin_e2e::framework::TestFramework;
+use crate::bitcoin_e2e::node::L2Node;
+use crate::bitcoin_e2e::test_case::{TestCase, TestCaseRunner};
+use crate::bitcoin_e2e::Result;
+
+struct MempoolAcceptTest;
+
+#[async_trait]
+impl TestCase for MempoolAcceptTest {
+    fn bitcoin_config() -> BitcoinConfig {
+        BitcoinConfig {
+            extra_args: vec![
+                // Restrictive mempool policy
+                "-limitancestorcount=0",
+                "-limitancestorsize=0",
+            ],
+            ..Default::default()
+        }
+    }
+
+    async fn run_test(&self, f: &TestFramework) -> Result<()> {
+        let sequencer = f.sequencer.as_ref().unwrap();
+        let da = f.bitcoin_nodes.get(0).expect("DA not running.");
+
+        let min_soft_conf_per_commitment = sequencer.min_soft_confirmations_per_commitment();
+
+        // publish min_soft_conf_per_commitment - 1 confirmations, no commitments should be sent
+        for _ in 0..min_soft_conf_per_commitment {
+            sequencer.client.send_publish_batch_request().await;
+        }
+        sequencer
+            .wait_for_l2_height(min_soft_conf_per_commitment, None)
+            .await;
+
+        da.generate(FINALITY_DEPTH + 1, None).await?;
+
+        // TODO find the right assertions here
+        // Should be either 2 or 0
+        // Before this PR and the addition of testmempoolaccept, first tx would go in and second would be rejected due to mempool policy set above
```

Comment:
my suggestion is 


first set up the mempool with normal configs

see 2 txs of seqeuncer commitment

generate that block

shutdown bitcoin node

reboot it with thise mempool policy args above

trigger a commitment again, wait 3 mins or something and ceckmempoolsize is 0

that way test knows the sequencer can send commitments, and can check it can't produce a seq com for 3 mins -- due to `testmempoolaccept` (not guaranteed because of our nonce thing, but good enough for this feature)